INSTALL_LINUX.md

```diff
@@ -180,13 +197,16 @@ Natron's nodes are contained in separate repositories. To use the default nodes,
 You'll find installation instructions in the README of both these repositories. Both openfx-misc and openfx-io have submodules as well.
 
 Plugins must be installed in /usr/OFX/Plugins on Linux
-Or in a directory named "Plugins" located in the parent directory where the binary lies, e.g.:
+Or in a directory named "Plugins/OFX/Natron" located in the parent directory where the binary lies, e.g.:
 
 
-    bin/
+    App/
```

Comment:
Why change the dir name to `App`? binaries are usually installed in a `bin` dir

Reply:
I didn't do a "make install", just a make and run natron from the build folder. 
The natron executable is in a ./App folder in that case on my computer.
(In fact the doc didn't suggest to do a make install)

I can revert that change (and the Plugins path ) and add text for the this specific use case (run natron from build folder) as the plugins path is important to set the way I found (Plugins/OFX/) ?

On my side I prefer running from build folder so I can have different build of natron on the same machine along with the official portable version.